ext/standard/info.c

```diff
@@ -680,21 +674,20 @@ PHPAPI zend_string *php_get_uname(char mode)
 	if (mode == 's') {
 		php_uname = "Windows NT";
 	} else if (mode == 'r') {
-		snprintf(tmp_uname, sizeof(tmp_uname), "%d.%d", dwWindowsMajorVersion, dwWindowsMinorVersion);
-		php_uname = tmp_uname;
+		// TODO Limit buffer size?
```

Comment:
No need to, since we're only formatting two integers here. Or is about performance? I don't think this is relevant; unlikely that users call `php_uname()` or such in a tight loop.